2-mandatory/3-financial-times.js

```diff
@@ -23,59 +35,74 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  let arr = [];
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    if (/[0-9]/g.test(allArticleTitles[i])) {
```

Comment:
I think the comment on the function was hinting that you use `for of` on the titles to iterate through and detect numbers, but a regex is definitely a more efficient solution :+1:

Reply:
Thanks alot